spec/pQueue.js

```diff
@@ -0,0 +1,122 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PriorityQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('PriorityQueue', () => {
+  it('exists', () => {
+    expect(PriorityQueue).to.be.a('function')
+  })
+
+  context('enqueue', (element, priority) => {
+    it('adds one element to the queue queue.', () => {
```

Comment:
it('adds one element to the queue queue.', () => {
^ this line is a little confusing